### ISSUE TITLE
How to test the LLIE results

### ISSUE THREAD

**Issue by savvaki** [GuoLanqing/ShadowDiffusion#19]
Good day, 

I have looked at the supplementary materials and I see your low light enhancement results. How can I use the pretrained model for low light enhancement?

Thank you

**Comment by GuoLanqing** [comment]
You can resume the SRD model and fine-tune the model on the low-light datasets. And use the all-one matrix mask and remove the mask refinement loss.
